tests/topotests/lib/common_config.py

```diff
@@ -3976,29 +3982,206 @@ def verify_vrf_vni(tgen, input_dict):
     logger.debug("Exiting lib API: {}".format(sys._getframe().f_code.co_name))
     return False
 
+def iperfSendIGMPJoin(tgen, server, bindToAddress, l4Type='UDP',
+                      join_interval=1, inc_step=0, repeat=0):
+    """
+    Run iperf to send IGMP join and traffic
+
+    Parameters:
+    -----------
+    * `tgen`  : Topogen object
+    * `l4Type`: string, one of [ TCP, UDP ]
+    * `server`: iperf server, from where IGMP join would be sent
+    * `bindToAddress`: bind to <host>, an interface or multicast
+                       address
+    * `join_interval`: seconds between periodic bandwidth reports
+    * `inc_step`: increamental steps, by default 0
+    * `repeat`: Repetition of group, by default 0
 
-def required_linux_kernel_version(required_version):
```

Comment:
Why the deletion of requied_linux_kernel_version?

Reply:
This kills a bunch of tests.  Did you even run this?

Reply:
I have rebased the code